### ISSUE TITLE
Failing NighwatchCI test

### ISSUE THREAD

**Issue by jacogr** [ethcore/parity-dapps-status#179]
https://travis-ci.org/ethcore/parity-dapps-status/jobs/133120088

```
Running:  Assert form
 ✔ Element <[data-test="Call-call-1"]> was visible after 383 milliseconds.
 ✔ Expected element <[data-test="Call-call-1"] > span:nth-child(1)> text to contain: "#1" - condition was met in 364ms
 ✖ Expected element <[data-test="Call-call-1"] > pre:nth-of-type(1)> text to contain: "ethcore_setExtraData(123)"  - expected "contain 'ethcore_setExtraData(123)'" but got: ethcore_setExtraData("123")
    at Object.Assert form (/home/travis/build/ethcore/parity-dapps-status/test/e2e/rpc-page.js:59:19)
    at Module.call (/home/travis/build/ethcore/parity-dapps-status/node_modules/nightwatch/lib/runner/module.js:63:34)
    at /home/travis/build/ethcore/parity-dapps-status/node_modules/nightwatch/lib/runner/testcase.js:97:29
    at _fulfilled (/home/travis/build/ethcore/parity-dapps-status/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/home/travis/build/ethcore/parity-dapps-status/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/home/travis/build/ethcore/parity-dapps-status/node_modules/q/q.js:796:13)
    at /home/travis/build/ethcore/parity-dapps-status/node_modules/q/q.js:556:49
    at runSingle (/home/travis/build/ethcore/parity-dapps-status/node_modules/q/q.js:137:13)
    at flush (/home/travis/build/ethcore/parity-dapps-status/node_modules/q/q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
```

@goldylucks Any possible ideas?

**Comment by goldylucks** [comment]
it will b fixed once #174 is merged

**Comment by jacogr** [comment]
Cool, glad to see you are on it. I was worried I missed a beat.
